packages/zoe/src/contractSupport/priceAuthority.js

```diff
@@ -259,7 +258,7 @@ export const makeOnewayPriceAuthorityKit = opts => {
   };
 
   /** @type {PriceAuthority} */
-  const priceAuthority = Far('PriceAuthority', {
+  const priceAuthority = Far('PriceAggregator', {
```

Comment:
why?

Reply:
There are multiple objects going by the name "priceAggregator", or being called priceAggregator, and when I see them in a debugger or trace statement, I'd like to be able to distinguish them.

Reply:
It does have some appeal. `PriceAuthority` is an interface, and `PriceAggregator` is an implementation of it. Being more specific about the implementation seems reasonable... even though the argument to `Far` is called **interface** in various contexts.

Reply:
I wrote `priceAggregator` when I meant `priceAuthority`. The latter is the name that is overused, and the former is a name that only applies to this instance.